install/kubernetes/cilium/templates/hubble-ui-deployment.yaml

```diff
@@ -40,6 +40,10 @@ spec:
 {{- with .Values.hubble.ui.tolerations }}
       tolerations:
       {{- toYaml . | trim | nindent 6 }}
+{{- end }}
+{{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.imagePullSecrets | indent 6 }}
```

Comment:
Any reason this is formatted differently than the `tolerations:` L41? Shouldn't this be indented to 8 instead of 6?

Reply:
I was basing it off: https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml#L88

As it's an array 6 indentation is still valid but happy to change to 8 if preferred 
```
      imagePullSecrets:
      - item
      
      imagePullSecrets:
        - item
```

Reply:
Thank you, I see that we indent `imagePullSecrets` inconsistently across the different Helm template, but that's outside the scope of this PR.